materials/dplyr.md

```diff
@@ -5,79 +5,102 @@ title: Working with Tabular Data
 language: R
 --- 
 
-## Packages
+> Remember to
+> 
+> * download [`surveys.csv`](https://ndownloader.figshare.com/files/2292172).   
+> * download [`portal_mammals.sqlite`](https://ndownloader.figshare.com/files/2292171).
+> * `setwd()` in R.
```

Comment:
Ok, that makes sense to me. This is a good time to point out that I don't use RStudio Projects or Git from RStudio and did not test those aspects of the `lectures/`. In these cases, my review was more conceptual.